returnn/training.py

```diff
@@ -287,6 +287,8 @@ def _get_run_cmd(self):
         return run_cmd
 
     def info(self):
+        import numpy as np
```

Comment:
what was the reasoning behind doing a local import vs module level?

Reply:
* I like to keep things local to see where they are used. That's a thing you should do in general, for all kinds of things, about the structure of code (e.g. local variables, etc.).
* It's not good practice to put too much things into the global namespace.
* It could increase the module load time when `numpy` was not imported yet.

Reply:
Note, the only (valid) argument to put the import at module level is that you see all the dependencies of the module more easily. But I'm not sure this is so important here.

Reply:
It comes very surprising to me to break with the convention now, but I do not care too much. 

>  the only

Definitely not the only in general, maybe the only one here. Once you use it more often you save lines of code, definitely a benefit.